### PR TITLE
[sovereign] Update mock notifier

### PR DESCRIPTION
**Pull request by mariusmihaic**
## Reasoning behind the pull request
- mock notifier update
- 
- 
  
## Proposed changes
- updated `deposit` logs
- updated `execute` logs
- 

## Testing procedure
- 
- 
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: cmd/sovereignnode/systemTestDemo/mockNotifier/vars.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 const (
 	addressLen = 32
 	hashSize   = 32
+	lenSize    = 4
```
Comment (mariusmihaic) [inline cmd/sovereignnode/systemTestDemo/mockNotifier/vars.go:13]: Len of what?

File: cmd/sovereignnode/systemTestDemo/mockNotifier/disabledGRPCServer.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+type disabledGRPCServer struct {
+}
+
+// NewDisabledGRPCServer -
+func NewDisabledGRPCServer() *disabledGRPCServer {
+	return &disabledGRPCServer{}
+}
+
+// Stop does nothing
```
Comment (mariusmihaic) [inline cmd/sovereignnode/systemTestDemo/mockNotifier/disabledGRPCServer.go:11]: Perhaps make comments consistent by either putting `-` everywhere or actually adding comments

Comment (mariusmihaic) [reply cmd/sovereignnode/systemTestDemo/mockNotifier/disabledGRPCServer.go:11]: Applies to `cmd/sovereignnode/systemTestDemo/mockNotifier/disabledMockGRPCServer.go` as well

File: cmd/sovereignnode/systemTestDemo/mockNotifier/flags.go [hunk 3]
```diff
@@ -14,4 +14,18 @@ var (
 			" log level.",
 		Value: "*:" + logger.LogTrace.String(),
 	}
+	grpcEnabled = cli.BoolFlag{
+		Name:  "grpc-enabled",
+		Usage: "Boolean option for enabling GRPC server.",
+	}
+	sovereignBridgeCertificateFile = cli.StringFlag{
+		Name:  "certificate",
+		Usage: "The path for sovereign outgoing bridge certificate file.",
```
Comment (mariusmihaic) [inline cmd/sovereignnode/systemTestDemo/mockNotifier/flags.go:23]: Here and below?
```suggestion
		Usage: "The path for sovereign grpc bridge service certificate file.",
```

File: cmd/sovereignnode/systemTestDemo/mockNotifier/interface.go [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package main
+
+type MockServer interface {
```
Comment (mariusmihaic) [inline cmd/sovereignnode/systemTestDemo/mockNotifier/interface.go:3]: Missing comms

File: cmd/sovereignnode/systemTestDemo/mockNotifier/notifier.go [hunk 5]
```diff
@@ -289,43 +298,89 @@ func createOutGoingBridgeOpsConfirmationLogs(confirmedBridgeOps []*ConfirmedBrid
...
+	eventData = append(eventData, numberToBytes(uint64(len(args)), lenSize)...)     // number of arguments
+	for _, arg := range args {
+		eventData = append(eventData, numberToBytes(uint64(len(arg)), lenSize)...) // length of current argument
+		eventData = append(eventData, arg...)                                      // current argument
+	}
+
+	return eventData
+}
+
+func numberToBytes(number uint64, size int) []byte {
```
Comment (mariusmihaic) [inline cmd/sovereignnode/systemTestDemo/mockNotifier/notifier.go:377]: I think we could've used bigEndianConverter here if that would work...
no strong opinion, can stay as it is

File: cmd/sovereignnode/systemTestDemo/mockNotifier/vars.go (hunk 1 above)
Comment (axenteoctavian) [reply cmd/sovereignnode/systemTestDemo/mockNotifier/vars.go:13]: For coding, first you encode the length, the you encode the item. So it's length of the item

File: cmd/sovereignnode/systemTestDemo/mockNotifier/flags.go (hunk 3 above)
Comment (mariusmihaic) [reply cmd/sovereignnode/systemTestDemo/mockNotifier/flags.go]: Missed fix for below variable : `sovereignBridgeCertificatePkFile`

File: cmd/sovereignnode/systemTestDemo/mockNotifier/vars.go (hunk 1 above)
Comment (mariusmihaic) [reply cmd/sovereignnode/systemTestDemo/mockNotifier/vars.go:13]: Yes, but please rename it :D

**Review by mariusmihaic — COMMENTED** [review]
:green_circle:

**Review by axenteoctavian — APPROVED** [review]

**Pull request merged**